.chloggen/sigv4_add_unmarshal.yaml

```diff
@@ -0,0 +1,16 @@
+# One of 'breaking', 'deprecation', 'new_component', 'enhancement', 'bug_fix'
+change_type: enhancement
+
+# The name of the component, or a single word describing the area of concern, (e.g. filelogreceiver)
+component: sigv4authextension
+
+# A brief description of the change.  Surround your text with quotes ("") if it needs to start with a backtick (`).
+note: Add Unmarshal() method to config.go
```

Comment:
Can we get a bit more information in this changelog? If this just an internal change, just remove this changelog and mark the PR as "Skip Changelog"